### ISSUE TITLE
Unable to render_graph

### ISSUE THREAD

**Issue by dashed** [saresend/Gust#3]
I tried to use gust with the following code:

```rust
extern crate gust;

use gust::backend::bar_chart::BarChart;
use gust::frontend::write::render_graph;
use gust::backend::general::FileType;

fn main() {
    println!("Hello, world!");

    let mut b = BarChart::new();

    let v = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L"];
    for i in 0..10 {
        b.add_data(v[i].to_string(), (i * i * i) as i32);
    }
    render_graph(&b, FileType::HTML).unwrap();
}
```

But it seems to panic at this line:

https://github.com/saresend/Gust/blob/38586147c0836795c69e5e8adc174149641cba91/src/frontend/html_render.rs#L16

**Comment by saresend** [comment]
I'll look into it, its possible Cargo isn't bundling the appropriate files when run. Should hopefully be fixed soon!

**Comment by saresend** [comment]
Alright, it should be fixed in version 0.1.4, just published! Just add `gust = "0.1.4"` to your cargo.toml file.
